plugin/core/windows.py

```diff
@@ -666,7 +666,7 @@ def __init__(self, manager: WindowManager, server_name: str) -> None:
         self._manager = ref(manager)
         self._server_name = server_name
 
-    def log(self, message: str, params: Any, log_payload: bool) -> None:
+    def log(self, message: str, params: Any, log_payload: Optional[bool] = True) -> None:
```

Comment:
The type of log_payloads should be `bool` and its default value should be `False`.

Reply:
Why default `False`? Then we have to pass `True` in most cases where now we can just skip it.
Do you want thing more explicit?

Reply:
Right, sorry, I think I need more coffee. I only have a problem with the Optional. That makes the type have 3 values whereas 2 possible values should be sufficient. The default should stay `True`.